recipes/motuclient/LICENSE

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
The license file is already in the source distribution, so you can remove this file.